pkgs/development/python-modules/channels-redis/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, redis, channels, msgpack, aioredis, hiredis, asgiref
+}:
+
+buildPythonPackage rec {
+  pname = "channels-redis";
+  version = "2.4.0";
+  src = fetchPypi {
+    inherit version;
+    pname = "channels_redis";
+    sha256 = "1g4izdf8237pwxn85bv5igc2bajrvck1p2a7q448qmjfznrbrk5p";
+  };
+  buildInputs = [ redis channels msgpack aioredis hiredis asgiref ];
```

Comment:
The following install requires should be `propagatedBuildInputs` 
* https://github.com/django/channels_redis/blob/master/setup.py#L34